lib/Field/styles.scss

```diff
@@ -412,7 +412,7 @@ $field-data-p-line-height: 1.688rem !default;
     }
   }
 
-  *:last-child {
+  *:last-child:not(button) {
```

Comment:
I also had a problem with the `.field.has-formatting` in that I _think_ it's only supposed to apply formatting to the content editor field bit, yet it is applying styling to the elements on the left (settings/validation/label). But maybe that's a problem outside your PR? 
See https://github.com/gathercontent/gather-ui/pull/727/files#diff-5871cfcb3e11ef3631cff523e1bce82cR316

Reply:
sadly my stuff is inside the field so will be effected either way!